commands/logout.go

```diff
@@ -105,16 +110,18 @@ func logout(args logoutArguments) error {
 	authHeader := "giantswarm " + args.token
 	logoutResponse, apiResponse, err := apiClient.UserLogout(authHeader, requestIDHeader, logoutActivityName, cmdLine)
 	if err != nil {
-		return fmt.Errorf("Error in API request to logout: %s", err.Error())
+		// special treatment for HTTP 401 (unauthorized) error,
+		// in which case no JSON body is returned.
+		if apiResponse.Response.StatusCode == http.StatusUnauthorized {
+			return microerror.Mask(notAuthorizedError)
+		}
+
+		// other cases
+		return microerror.Maskf(unspecifiedAPIError, err.Error())
 	}
 
 	if logoutResponse.StatusCode != apischema.STATUS_CODE_RESOURCE_DELETED {
-		if apiResponse.Response.StatusCode == http.StatusUnauthorized {
-			// we ignore a 401 (Unauthorized) response here, as it means in most cases
-			// that the token submitted was already expired.
-			return nil
-		}
-		return fmt.Errorf("Error in API request to logout: %#v", logoutResponse)
+		return microerror.Maskf(unspecifiedAPIError, fmt.Sprintf("Error in API request to logout: %v", logoutResponse))
```

Comment:
Your error message will look like: `unspecified API error: Error in API request to logout: ...`. I'd rather make it something like`unspecified API error: requesting to logout: ...`. That would also solve the problem of capital letter. WDYT?

Reply:
We talked about it. Just to reference: `return microerror.Maskf(unspecifiedAPIError, "requesting logout: %v", logoutResponse)`